scripts/barclamp_mgmt_lib.rb

```diff
@@ -260,6 +260,8 @@ def generate_assets_manifest
 
   manifests = Pathname.new(CROWBAR_PATH).join("barclamps", "manifests")
 
+  manifests.mkdir unless manifests.exist?
```

Comment:
No, what I mean when I run the barclamp_install.rb in my dev setup the first time (the folder doesn't exist) the it fail due to the manifest can not written. I need to create the folder by hand before running the script then it works.

This is just the solution which I use since some weeks...